tests/e2e/specs/themes/themes.test.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	installTheme,
+	deleteTheme,
+	isThemeInstalled,
+	__experimentalRest as rest,
+} from "@wordpress/e2e-test-utils";
+
+const themeSlug = "hello-elementor";
```

Comment:
Can we replace this with a skeleton theme that has [only the files required for a theme to be considered valid](https://developer.wordpress.org/themes/releasing-your-theme/required-theme-files/)?